### PR TITLE
Add note about rails-deprecated-sanitizer to Upgrade Guide.

### LINKED ISSUE #25629: 5.0 Upgrade Guide should include support dropping for rails-deprecated_sanitizer
[`rails-deprecated_sanitizer`'s README](https://github.com/rails/rails-deprecated_sanitizer) says it will be supported until Rails 5.0, which is now released. 

The [`activerecord-deprecated-finders` gem had a mention in the Rails 5 upgrade guide](http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#removed-support-for-activerecord-deprecated-finders-gem), shouldn't this get one too?

### PR DESCRIPTION
**Pull request by connorshea**
Title says it all.

Fixes #25629.

### CODE REVIEW

File: guides/source/upgrading_ruby_on_rails.md [hunk 1]
```diff
@@ -237,6 +237,10 @@ The `protected_attributes` gem is no longer supported in Rails 5.
 
 The `activerecord-deprecated_finders` gem is no longer supported in Rails 5.
 
+### Removed support for `rails-deprecated_sanitizer` gem
+
+The `rails-deprecated_sanitizer` gem is no longer supported in Rails 5, the [rails-html-sanitizer](https://github.com/rails/rails-html-sanitizer) gem should be used instead.
```
Comment (rafaelfranca) [inline guides/source/upgrading_ruby_on_rails.md]: Thank you for the pull request but I'll support this gem since we are still using in Shopify.

Comment (connorshea) [inline guides/source/upgrading_ruby_on_rails.md]: @rafaelfranca you will probably want to update the gem's README in that case, feel free to close :)

Comment (rafaelfranca) [inline guides/source/upgrading_ruby_on_rails.md]: :+1: I still didn't get to work on that gem, but I'll update the README now.

**Pull request closed without merging**